docs/source/miscellaneous/stubs.rst

```diff
@@ -2,282 +2,357 @@
  Stubs
 #######
 
-Stub files are pieces of plain PHP code which only contain declarations without actually runnable
-code. A very basic stub looks like this:
+Stub files are pieces of PHP code which only contain declarations. They do not
+include runnable code, but instead contain empty function and method bodies.
+A very basic stub looks like this:
 
 .. code:: php
 
-   /** @var string */
-   const FOO = "foo";
-   /** @var int */
-   const BAR = "bar";
+    <?php
```

Comment:
I know this sounds silly, but the reStructuredText format requires indentation of 3 spaces... You may set up rstfmt 
 (https://github.com/php/php-src/tree/master/docs#formatting) which can automatically fix some of the formatting issues.

Reply:
That sounds indeed silly, as it makes pasting in code a real pain in the arse.

I've ran `rstfmt -w 100` now, so that should fix it.

Reply:
Hmm, TBH I don't know the answer about your question, but Ilija will surely be able to do so.

Reply:
Most of my blocks that I have in all my articles do:

```
This example::

    <?php
    // four identations
```

So not with ``.. code:`` — maybe that's the difference.